spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'Home page' do 
+  context 'authorized user' do
+    scenario 'can see other users successfully' do
+      user1 = FactoryGirl.create(:user)
+      user2 = FactoryGirl.create(:user)
+      sign_in_as(user2)
+      visit root_path
+
+      expect(page).to have_content(user1.email)
+      expect(page).to have_content(user2.email)
+    end
+  end
+
+  context 'visitor' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.